### PR TITLE
Feat/devsu 1748 split detected alt from tgr

### PR DESCRIPTION
**Pull request by kttkjl**
Preview

image

can be seen on `report/2a916519-4b6f-468c-8e05-0e992619c07f/kb-matches` on iprstaging

### CODE REVIEW

**Comment by kttkjl** [comment]
@areisle In terms of whether or not this is correct in the sense of data display, I have confirmed with Dustin [here](https://www.bcgsc.ca/jira/browse/DEVSU-1748?focusedCommentId=1560428&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1560428) this is the intended result

**Comment by areisle** [comment]
@kttkjl I'm unable to view that ticket due to permissions restrictions. could you summarize what the required changes described in the ticket are on this PR somewhere? otherwise I'll have to wait until I get those permissions to start reviewing this

**Comment by kttkjl** [comment]
Hi @areisle the changes required was for the table

The task at hand is 
Separate 'Detected Alterations From Targeted Gene Report' into two tables. One for somatic probes and one for germline probes, and rename to:
- 'Detected Alterations from Pharmacogenomic and Cancer Predisposition Targeted Gene Report'
- 'Detected Alterations From Somatic Targeted Gene Report'

The API call to get the data is via `/reports/${report.ident}/probe-results`
